js-api-spec/value/function.test.ts

```diff
@@ -80,3 +80,45 @@ describe('rejects a function signature that', () => {
   it('has no closing parenthesis', () => rejectsSignature('foo('));
   it('has a non-identifier name', () => rejectsSignature('$foo()'));
 });
+
+it('rejects a compiler function from a different compilation', () => {
+  let plusOne: Value | undefined;
+  compileString(
+    `
+    @use 'sass:meta';
+
+    @function plusOne($n) {@return $n + 1}
+    a {b: meta.call(foo(meta.get-function('plusOne')), 2)}
+  `,
```

Comment:
Style nit: Indent multiline strings 2 spaces beneath the opening quote, like so:
```suggestion
      @use 'sass:meta';

      @function plusOne($n) {@return $n + 1}
      a {b: meta.call(foo(meta.get-function('plusOne')), 2)}
    `,
```